### PR TITLE
Adds warning message for when a transaction with a non-default mixin …

### PR DESCRIPTION
**Pull request by skftn**
…value is created

### CODE REVIEW

File: main.qml [hunk 1]
```diff
@@ -608,18 +608,23 @@ ApplicationWindow {
         } else {
             console.log("Transaction created, amount: " + walletManager.displayAmount(transaction.amount)
                     + ", fee: " + walletManager.displayAmount(transaction.fee));
-
+            
+
```
Comment (moneromooo-monero) [inline main.qml]: Random spurious whitespace change

File: components/StandardDialog.qml [hunk 2]
```diff
@@ -100,6 +100,7 @@ Rectangle {
         RowLayout {
             id: column
             //anchors {fill: parent; margins: 16 }
+            Layout.topMargin: 14
```
Comment (moneromooo-monero) [inline components/StandardDialog.qml]: Seems unrelated

Comment (skftn) [reply components/StandardDialog.qml]: Fixed.

File: main.qml (hunk 1 above)
Comment (skftn) [reply main.qml]: Fixed.
